frontend/src/components/navigation/index.ts

```diff
@@ -1,2 +1,2 @@
-export { default as MenuItem } from './MenuItem';
-export * from './MenuItem';
+export { default as MenuItem } from './menuItem';
```

Comment:
Nazwa pliku menuItem powinna zaczynać się z dużej litery. Jest to komponent React'a i stąd taki wymóg.